index.html

```diff
@@ -2478,73 +2480,76 @@ <h3>Zero-Knowledge Proofs</h3>
 personally identifiable information contained on the degree.
         </p>
         <p>
-The key capabilities introduced by zero-knowledge proof mechanisms are:
+The key capabilities introduced by zero-knowledge proof mechanisms are the
+ability of a <a>holder</a> to:
         </p>
-        <ol>
+
+        <ul>
           <li>
-The ability of a <a>holder</a> to combine multiple <a>verifiable credentials</a>
-from multiple <a>issuers</a> into a single presentation without revealing
-<a>credential</a> or <a>subject</a> identifiers to the <a>verifier</a>. This
-makes it more difficult for the <a>verifier</a> to collude with any of the
-issuers regarding the issued <a>credentials</a>.
+Combine multiple <a>verifiable credentials</a> from multiple <a>issuers</a> into
+a single <a>verifiable presentation</a> without revealing
+<a>verifiable credential</a> or <a>subject</a> identifiers to the
+<a>verifier</a>. This makes it more difficult for the <a>verifier</a> to collude
+with any of the issuers regarding the issued <a>verifiable credentials</a>.
           </li>
           <li>
-The ability of a <a>holder</a> to selectively disclose the claims in a <a>credential</a>
-to a <a>verifier</a> without requiring the issuance of multiple atomic <a>credentials</a>.
-This allows a <a>holder</a> to provide a <a>verifier</a> with precisely the information
-they need and nothing more.
+Selectively disclose the claims in a <a>verifiable credential</a> to a
+<a>verifier</a> without requiring the issuance of multiple atomic
+<a>verifiable credentials</a>. This allows a <a>holder</a> to provide a
+<a>verifier</a> with precisely the information they need and nothing more.
           </li>
           <li>
-The ability of a <a>holder</a> to produce a derived <a>credential</a> that is formatted
-according to the <a>verifier</a>'s data schema rather than the <a>issuer</a>'s, without
-needing to involve the <a>issuer</a> after <a>credential</a> issuance. This provides
-a great deal of flexibility for <a>holders</a> to use the <a>credentials</a> they have
-been issued.
+Produce a derived <a>verifiable credential</a> that is formatted according to
+the <a>verifier's</a> data schema rather than the <a>issuer's</a>, without
+needing to involve the <a>issuer</a> after <a>verifiable credential</a>
+issuance. This provides a great deal of flexibility for <a>holders</a> to use
+the <a>verifiable credentials</a> they were issued with.
           </li>
         </ol>
+
         <p>
-This specification describes a data model that supports zero-knowledge proof mechanisms.
-The examples below highlight how the data model may be used to issue,
-present, and verify zero-knowledge <a>verifiable credentials</a>.
+This specification describes a data model that supports zero-knowledge proof
+mechanisms. The examples below highlight how the data model may be used to
+issue, present, and <a>verify</a> zero-knowledge <a>verifiable credentials</a>.
```

Comment:
Not sure why verify becomes <a>verify</a> when issue and present are left as plain text.

Reply:
The above comment does not show the "&lta&gt"verify"&lt/a&gt".

Reply:
Fixed.

Reply:
Yes, I guessed that. 😄